docs/src/main/sphinx/client/cli.rst

```diff
@@ -16,6 +16,11 @@ It can be used with Java version 8 and higher.
 The CLI uses the :doc:`Trino client REST API </develop/client-protocol>` over
 HTTP/HTTPS to communicate with the coordinator on the cluster.
 
+The CLI version should be identical to the version of the Trino cluster, or
+newer. Older versions typically work, but only a subset is regularly tested.
```

Comment:
I feel like this first sentence is awkward. "The CLI version should be up-to-date with or newer than the Trino cluster version" might be a little smoother? Or something along those lines.

Reply:
... I inserted this and reread ... "up-date-with or newer" feels worse than what there is in the PR now.

Reply:
Maybe just replace "identical" with "up-to-date"?